samples/watchdog/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _watchdog_sample:
+
+Watchdog Timer
+##############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Watchdog Timer sample demonstrates how to configure a PMIC TIMER periphery in the Watchdog Timer mode with the use of npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses the Timer driver to do the following:
+
+  * Enter and configure the TIMER periphery in the Watchdog Timer mode.
+  * Register a handler to be called when the watchdog event occurs.
```

Comment:
```suggestion
  * Register a handler to be called when the WATCHDOG event occurs.
```
To be confirmed if watchdog is an event in this case.

Reply:
Yes